hie.yaml.cbl

```diff
@@ -1,7 +1,9 @@
 # This is a sample hie.yaml file for opening haskell-language-server
 # in hie, using cabal as the build system.  To use is, copy it to a
 # file called 'hie.yaml'
-
+# WARNING: This configuration does not currently work.
```

Comment:
I think this should be

```
# WARNING: This configuration does not currently work in ghcide, but does in hie
```
And ditto for the stack one

Reply:
oh, right, i forgot hie support it 😄 
I wanted to stress that those files do not work with hls itself, just in case someone tries to use them.